.github/workflows/live-deploy.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       - uses: actions/checkout@v2
 
       - name: Setup Node
-        uses: actions/setup-node@v2.1.0
```

Comment:
I don't think `2.3.0` exists. See the releases page: https://github.com/actions/setup-node/releases

Furthermore, check out the comparison from the latest semantic release and `v2-beta`: https://github.com/actions/setup-node/compare/v2-beta...v2.1.3

It looks like we can use `v2-beta` to stay on the latest v2 changes w/o having to worry about minor versions.

Reply:
Ooops, I meant 2.1.3

Reply:
Ok